src/interfaces/dataStoredInToken.ts

```diff
@@ -0,0 +1,5 @@
+interface DataStoredInToken {
```

Comment:
```suggestion
interface DataInToken {
```

popraw nazwe pliku na `DataInToken.interface.ts`

Reply:
OK